src/common/src/session_config/mod.rs

```diff
@@ -22,14 +22,15 @@ use crate::error::{ErrorCode, RwError};
 
 // This is a hack, &'static str is not allowed as a const generics argument.
 // TODO: refine this using the adt_const_params feature.
-const CONFIG_KEYS: [&str; 7] = [
+const CONFIG_KEYS: [&str; 8] = [
     "RW_IMPLICIT_FLUSH",
     "QUERY_MODE",
     "RW_FORCE_DELTA_JOIN",
     "EXTRA_FLOAT_DIGITS",
     "APPLICATION_NAME",
     "DATE_STYLE",
     "RW_BATCH_ENABLE_LOOKUP_JOIN",
+    EXPLAIN_VERBOSE_STRING,
```

Comment:
According to pg's doc, verbose is an option of explain, so why we make it  a session config?